build.gradle

```diff
@@ -23,8 +23,8 @@ repositories {
   }
 }
 
-sourceCompatibility = 1.6
-targetCompatibility = 1.6
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
```

Comment:
Not my decision, but I highly doubt the project owner will merge if indeed the change requires Java 8.